apps/mochi-web/components/explore/index/sections/TransactionSection.tsx

```diff
@@ -41,40 +41,39 @@ export const TransactionSection = () => {
   const isLoading = loading || !txnsCurrentPage
 
   return (
-    <div className="overflow-auto">
-      <div ref={containerRef} className="px-6 mx-auto" style={{ width: 1440 }}>
-        <div className="flex justify-end py-2 gap-4">
-          <ChainPicker
-            value={filters.chainId || ''}
-            onChange={(chainId) => setFilters({ chainId })}
-          />
-          <PlatformPicker
-            value={filters.platform || ''}
-            onChange={(platform) => setFilters({ platform })}
-          />
-        </div>
-        <TransactionTable
-          loadingRows={10}
-          data={txnsCurrentPage}
-          isLoading={isLoading}
-          onRow={(tx) => {
-            return {
-              onClick: () => {
-                window.open(ROUTES.TX_RECEIPTS(tx.code))
-              },
-            }
-          }}
-          componentsProps={{
-            pagination: {
-              initalPage: 1,
-              initItemsPerPage: DEFAULT_PAGE_SIZE,
-              totalItems: total,
-              onItemPerPageChange: setSize,
-              onPageChange: setPage,
-            },
-          }}
+    <div ref={containerRef} className="mx-auto" style={{ maxWidth: 1488 }}>
+      <div className="flex justify-end py-2 gap-4 px-6">
+        <ChainPicker
+          value={filters.chainId || ''}
+          onChange={(chainId) => setFilters({ chainId })}
+        />
+        <PlatformPicker
+          value={filters.platform || ''}
+          onChange={(platform) => setFilters({ platform })}
         />
```

Comment:
Can we reset the page number to 1 after doing either of the filters?

Reply:
That logic is already built-in in the store:

<img width="610" alt="image" src="https://github.com/consolelabs/mochi-ui/assets/11691985/6bfe4bd9-169d-49b2-97c4-a08b7780ce0b">